Validation/RecoTau/python/dataTypes/ValidateTausOnRealElectronsData_cff.py

```diff
@@ -120,8 +120,23 @@
 proc.TauValNumeratorAndDenominatorRealElectronsData.visit(zttModifier)
 #-----------------------------------------
 
+#Set discriminators
+discs_to_retain = ['ByDecayModeFinding', 'MVA6']
```

Comment:
it will be better to use something more general than 'MVA6' to select against-electron discrimiants, e.g. "ElectronRejection"?